Embedded Metadata.js

```diff
@@ -766,25 +766,27 @@ function getAuthorFromByline(doc, newItem) {
 
 function finalDataCleanup(doc, newItem) {
 	/**If we already have tags - run through them one by one,
-	 * split where ncessary and concat them.
-	 * This  will deal with multiple tags, some of them comma delimited,
+	 * split where necessary and concat them.
+	 * This will deal with multiple tags, some of them comma delimited,
 	 * some semicolon, some individual
 	 */
-	if (newItem.tags.length && Zotero.parentTranslator) {
-		var tags = [];
-		for (var i in newItem.tags) {
-			newItem.tags[i] = newItem.tags[i].trim();
-			if (newItem.tags[i].indexOf(';') == -1) {
-				//split by comma, since there are no semicolons
-				tags = tags.concat( newItem.tags[i].split(/\s*,\s*/) );
-			} else {
-				tags = tags.concat( newItem.tags[i].split(/\s*;\s*/) );
+	if (newItem.tags && newItem.tags.length && Zotero.parentTranslator) {
```

Comment:
> with the thinking that if there are multiple tags, the comma is likely not a delimiter

Yes, that would be another possibility. However, the current code does explicitly split on comma in individual tags, when there are more than one. Also the comment above that code says:

> This will deal with multiple tags, some of them comma delimited, some semicolon, some individual

This code is several years old, such that I don't know whether there are use cases for this, but there might be. It is a little hard to predict, which consequences changes in EM translator have. Thus, I decided not change "a running system".

Ideally, we could test all examples in EM translator and its depending translators as well, but this is AFAIK not easily possible.